src/patternfly/components/Tooltip/tooltip.scss

```diff
@@ -68,8 +66,8 @@
 .pf-c-tooltip__arrow {
   position: absolute;
   width: var(--pf-c-tooltip__arrow--Width);
-  height: var(--pf-c-tooltip__arrow--Height);
+  height: var(--pf-c-tooltip__arrow--Width);
   pointer-events: none;
-  background-color: var(--pf-c-tooltip__arrow--BackgroundColor);
-  box-shadow: var(--pf-c-tooltip__arrow--BoxShadow);
+  background-color: var(--pf-c-tooltip__content--BackgroundColor);
```

Comment:
I think the variable name should still have arrow in it, but then assign `var(--pf-c-tooltip__content--BackgroundColor);` to `--pf-c-tooltip__arrow--BackgroundColor` in the head

Reply:
In the effort of consolidating variables, I don't think that's necessary. I've talked to @mceledonia about these cases. As far as patternfly is concerned, I think the arrow's background color and box shadow should always match the tooltip content, meaning it's presented as a single thing with the same style. I would argue that we should make it so that if you update the variable to change either one, it changes the other, and we don't need to give the end user the flexibility of having a unique variable they can use to make the background or box shadow for the arrow different from the content. Also this is less code. What do you think?

Reply:
Okay that sounds good to me!